### PR TITLE
Hong Kong - Information Technology was replaced by Technology and Innovation

### PR DESCRIPTION
**Pull request by MarcinMisztalTR**
Hello,
In 2021 "Information Technology" was replaced by "Technology and Innovation" in Hong Kong

Links:
https://en.wikipedia.org/wiki/Information_Technology_(constituency)
https://en.wikipedia.org/wiki/Technology_and_Innovation_(constituency)

### CODE REVIEW

**Review by HKSenior — CHANGES_REQUESTED** [review]

File: identifiers/country-hk/functional_constituencies.csv [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
-ocd-division/country:hk/functional_constituency:financial_services,Financial Services
-ocd-division/country:hk/functional_constituency:sports_performing_arts_culture_and_publication,"Sports, Performing Arts, Culture and Publication"
-ocd-division/country:hk/functional_constituency:import_and_export,Import and Export
-ocd-division/country:hk/functional_constituency:textiles_and_garment,Textiles and Garment
-ocd-division/country:hk/functional_constituency:wholesale_and_retail,Wholesale and Retail
-ocd-division/country:hk/functional_constituency:information_technology,Information Technology
-ocd-division/country:hk/functional_constituency:catering,Catering
-ocd-division/country:hk/functional_constituency:district_council_first,District Council (First)
-ocd-division/country:hk/functional_constituency:district_council_second,District Council (Second)
+id,name,validFrom,validTo
```
[the remark this answers is not in the document]
Comment (MarcinMisztalTR) [reply identifiers/country-hk/functional_constituencies.csv]: Hello,
Thank you for your response. I've renamed "validTo" to "validThrough" for both Hong Kong files (functional_constituencies.csv and geographical_constituencies.csv) to not mislead with using validTo in the future. Also recompiled country-hk.csv file

**Review by HKSenior — APPROVED** [review]
Thanks for making the corresponding changes.

File: identifiers/country-hk/functional_constituencies.csv [hunk 2]
```diff
@@ -1,30 +1,31 @@
...
+ocd-division/country:hk/functional_constituency:industrial_first,Industrial (First),,
+ocd-division/country:hk/functional_constituency:industrial_second,Industrial (Second),,
+ocd-division/country:hk/functional_constituency:finance,Finance,,
+ocd-division/country:hk/functional_constituency:financial_services,Financial Services,,
+ocd-division/country:hk/functional_constituency:sports_performing_arts_culture_and_publication,"Sports, Performing Arts, Culture and Publication",,
+ocd-division/country:hk/functional_constituency:import_and_export,Import and Export,,
+ocd-division/country:hk/functional_constituency:textiles_and_garment,Textiles and Garment,,
+ocd-division/country:hk/functional_constituency:wholesale_and_retail,Wholesale and Retail,,
+ocd-division/country:hk/functional_constituency:information_technology,Information Technology,2021,
+ocd-division/country:hk/functional_constituency:technology_and_innovation,Technology and Innovation,,2021
```
Comment (jloutsenhizer) [inline identifiers/country-hk/functional_constituencies.csv:28]: I believe the timeline for the abolishing of the old constituency and for the forming of the new constituency would be the same as for the geographical constituencies (See PR #272).

Please update these to mach the `2021-12-19` date

**Review by jloutsenhizer — CHANGES_REQUESTED** [review]
One nitpick about the dates used and then I think this is good.

File: identifiers/country-hk/functional_constituencies.csv (hunk 2 above)
Comment (MarcinMisztalTR) [reply identifiers/country-hk/functional_constituencies.csv]: Hello, thank you for the response, I've updated the date from 2021 to 2021-12-19

**Review by jloutsenhizer — APPROVED** [review]

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**